pg_bm25/sql/_bootstrap_quickstart.sql

```diff
@@ -1,55 +1,63 @@
-CREATE TABLE paradedb.mock_items (
-    id SERIAL PRIMARY KEY,
-    description TEXT,
-    rating INTEGER CHECK (
-        rating BETWEEN 1
-        AND 5
-    ),
-    category VARCHAR(255),
-    in_stock BOOLEAN,
-    metadata JSONB
-);
+DO $$
+BEGIN
+    IF NOT EXISTS (SELECT FROM pg_catalog.pg_tables 
+                   WHERE schemaname = 'paradedb' AND tablename = 'mock_items') THEN
+        CREATE TABLE paradedb.mock_items (
+            id SERIAL PRIMARY KEY,
+            description TEXT,
+            rating INTEGER CHECK (
+                rating BETWEEN 1
+                AND 5
+            ),
+            category VARCHAR(255),
+            in_stock BOOLEAN,
+            metadata JSONB
+        );
```

Comment:
Why the procedural code instead of CREATE TABLE IF NOT EXISTS?

Reply:
Because we also don't want to run the subsequent INSERT commands if the table already exists